graphdatascience/tests/unit/conftest.py

```diff
@@ -114,9 +113,16 @@ def gds(runner: CollectingQueryRunner) -> Generator[GraphDataScience, None, None
 
 
 @pytest.fixture
-def aura_gds(runner: CollectingQueryRunner) -> Generator[AuraGraphDataScience, None, None]:
+def aura_gds(runner: CollectingQueryRunner, mocker: MockerFixture) -> Generator[AuraGraphDataScience, None, None]:
+    mocker.patch("graphdatascience.query_runner.neo4j_query_runner.Neo4jQueryRunner.create", return_value=runner)
+    mocker.patch(
```

Comment:
ah thats how you got around it. Could be an idea as well for the other unit tests.
But also increases the test-runtime to patch a lot